app/tools/android/make_apk.py

```diff
@@ -142,12 +140,30 @@ def FindExtensionJars(root_path):
         extension_jars.append(extension_jar)
   return extension_jars
 
+# try to parse out suitable app_versionCodeBase based on app_version,
```

Comment:
try -> Try, please capital first letter for the beginning of sentence. And apply this guideline everywhere in this PR.